### ISSUE TITLE
[BUG] Function disappears if you give it an invalid header and try to save it

### ISSUE THREAD

**Issue by dyavc** [Dyalog/link#109]
**Describe the bug**

**To Reproduce**
]link.Create # d:/git/test2/
Linked: # ←→ d:\git\test2

      )fns
Virgo   affErreur       export  export2 init
)ed init
⍝⍝ and make its header line invalid like init;a;b; 
⍝⍝ and hit escape

press the fix as code in workspace and file button
get a Can't fix message box
now function has disappeared:

      )fns
Virgo   affErreur       export  export2

**Expected behaviour**
Expected that the the function would still be in the workspace, just as it was before starting the editor.

**Please complete the following information:**
Dyalog on Windows 17.1.37551 64-bit Unicode

 - Link branch and commit: [e.g. master 128292f, 17.1 c293d59]
not sure

]uversion link.create
framework:  2.4                                                                      
command:    ]LINK.Create                                                             
source:     C:\Program Files\Dyalog\Dyalog APL-64 17.1 Unicode\SALT\spice\link.dyalog
version:    2.12                                                                     
revision:                                                                            
commit:     2019 10 18 Adam: Fix ]Add help

**Comment by nicolas-dyalog** [comment]
Fixed in Link v2.1.
The invalid source code is flushed to file, and the APL function remains untouched because the Fix fail. Then on editing the function again, the editor reads back from file, showing the incorrect source code. The best of both worlds =)

**Comment by nicolas-dyalog** [comment]
duplicate of issue #143
